relationships/synthesis/AWS-RDS-CLUSTER-to-RDS-INSTANCE.yml

```diff
@@ -0,0 +1,26 @@
+relationships:
+  - name: awsRdsClusterContainsRdsInstance
+    version: 1
+    origins:
+      - AWS Integration
+    conditions:
+      - attribute: eventType
+        anyOf: [ "MetricRaw" ]
+      - attribute: entity.type
+        anyOf: [ "AWS_RDS_DB_INSTANCE" ]
+    relationship:
+      expires: P75M
+      relationshipType: CONTAINS
+      source:
+        lookupGuid:
+          candidateCategory: AWSRDSDBCLUSTER
+          fields:
+            - field: awsAccountId
+              attribute: aws.accountId
+            - field: awsRegion
+              attribute: aws.region
+            - field: awsRdsClusterIdentifier
+              attribute: aws.rds.dbClusterIdentifier
+      target:
+        extractGuid:
+          attribute: entity.guid
```

Comment:
Given this is infra and guid's will have NA on them we should add the type here.

See an example: https://github.com/newrelic/entity-definitions/blob/5045dfc1dfbf5bc8d34f671bf0fefec9e513c2c8/relationships/synthesis/INFRA-KUBERNETESCLUSTER-to-INFRA-KUBERNETES_STATEFULSET.yml#L27-L30
you can also use `attribute: attributeName`

Reply:
shouldn't this be covered by https://github.com/newrelic/entity-definitions/pull/1146/files#diff-206f20614e1d3907d473aeda80a67f065afec57a9ce559fb8363a5b959f9f997R9 ?